amen/audio.py

```diff
@@ -108,10 +108,13 @@ def _get_beats(self):
         tempo, beat_frames = librosa.beat.beat_track(
             y=y_mono, sr=self.analysis_sample_rate, trim=False)
 
-        # convert frames to times
-        beat_times = librosa.frames_to_time(beat_frames, sr=self.sample_rate)
```

Comment:
This line turned out to be the culprit.  I think this was only obvious when the track was long enough for the difference between sample_rate and analysis_sample_rate to be observable.  +1 for realistic test data...